### DISCUSSION TITLE
Parsing an encrypted itunes backup

### DISCUSSION THREAD

**Discussion by kkttc** [abrignoni/iLEAPP#504] [Q&A]
Hi, am I correct in that ileapp doens't have the ability to prompt for the password of an encrypted itunes backup?  If that is the case, is there a free or commercial tool that I can use to decrypt an itunes backup that I have extracted from the tar.gz?

Thanks.

**Comment by stark4n6** [comment]
You can try Jack Farley's script, works for recreating the folder structures and can handle encrypted backups if you know the password.

[https://github.com/jfarley248/iTunes_Backup_Reader](https://github.com/jfarley248/iTunes_Backup_Reader)

After running his script you can then run iLEAPP against the recreated folder structure to parse some things, hope that helps.
